### PR TITLE
Merge kie-dmn into drools

### PR DESCRIPTION
**Pull request by tarilabs**

### CODE REVIEW

**Comment by tarilabs** [comment]
Similar to #1171 but using "filter-branch" script:

```
# Using a temp directory
mkdir 20170328-merge
cd 20170328-merge/

# Cloning a copy of the repos only for the purpose of this work
git clone git@github.com:kiegroup/kie-dmn.git
git clone git@github.com:kiegroup/drools.git

# Working on "source" repo:
cd kie-dmn/
git filter-branch --index-filter \
      'git ls-files -s | sed "s-\t\"*-&kie\-dmn/-" |
       GIT_INDEX_FILE=$GIT_INDEX_FILE.new \
       git update-index --index-info &&
       mv "$GIT_INDEX_FILE.new" "$GIT_INDEX_FILE"
      ' HEAD
# it will move all the content into additional kie-dmn directory, which is the structure we need for drools repo

# Working on "destination" repo:
# Create new branch, and pull the changes from the "updatd" kie-dmn repo
cd ..
cd drools/
git remote add kie-dmn ~/20170328-merge/kie-dmn
git branch tarilabs-20170328-mergedmn
git checkout tarilabs-20170328-mergedmn
git pull kie-dmn master

# .. some additional POMs alignments ...

# ... making some local test:
mvn clean verify -pl :kie-dmn-validation -am

# The new branch on the destination repo is now ready for PR.
```

:warning: do NOT squash :warning:

**Comment by tarilabs** [comment]
```
Test Result (1 failure / +1)
org.drools.compiler.test.PositionalTest.testPositionalWithNull
```

Jenkins test again :8ball: 

:warning: do NOT squash :warning:

**Comment by tarilabs** [comment]
cc/ @etirelli @psiroky similarly to #1171 of yesterday, but with the script strategy as described in comments.

:warning: do NOT squash :warning:

**Comment by tarilabs** [comment]
(on my side is the double-green as yesterday as on #1171, but I hope this time GitHub doesn't do incoherent result depending on the viewer...)
image

File: kie-dmn/pom.xml [hunk 1]
```diff
@@ -34,8 +35,8 @@
   </repositories>
```
Comment (psiroky) [inline kie-dmn/pom.xml]: I think we can remove this `<repositories>` as they are inherited from the parent.

File: kie-dmn/pom.xml [hunk 2]
```diff
@@ -34,8 +35,8 @@
...
 
   <scm>
-    <connection>scm:git:https://github.com/kiegroup/kie-dmn.git</connection>
-    <developerConnection>scm:git:git@github.com:kiegroup/kie-dmn.git</developerConnection>
+    <connection>scm:git:https://github.com/kiegroup/drools.git</connection>
+    <developerConnection>scm:git:git@github.com:kiegroup/drools.git</developerConnection>
     <url>https://github.com/kiegroup/kie-dmn</url>
   </scm>
```
Comment (psiroky) [inline kie-dmn/pom.xml]: I would remove the `<scm>` completely as it is now inherited from the parent.

**Review by psiroky — APPROVED** [review]
Looks good, just two minor changes are the POM metadata. Thanks @tarilabs!

**Comment by psiroky** [comment]
I will merge this manually later today to avoid the additional merge commit the GitHub UI would introduce.

**Comment by etirelli** [comment]
looks good! Thank you both!

**Comment by psiroky** [comment]
Merged. I had to re do the git magic locally as I had to get up-to-date with the current master to avoid the merge confict. Thanks @tarilabs!

**Pull request closed without merging**